css/style.css

```diff
@@ -0,0 +1,1248 @@
+  padding-top: 1em;
+  padding-bottom: 1em;
+  background: #fd6632;
+}
+
+.header-logo-text strong {
+  font-weight: normal;
+}
+
+.curve-underline {
+  padding-bottom: .1em;
+  background-size: cover;
+  background: url("/img/orange-underline.svg") no-repeat;
+  background-position: bottom;
+}
+
+@media screen and (min-width: 50em) {
+  h1 {
+    font-size: 2.666em;
+  }
+}
+
+
+/*
+SERVICES COMPONENTS
+*/
+
+.services-container {
+  padding: 2.5em .5em 1em;
+  text-align: center;
+  background-color: #fd6632;
+}
+
+.services-text,
+.contact-text {
+  display: inline-block;
+  padding: .5em 0;
+  line-height: 1.4;
+}
+
+.services-category {
+  padding: 1.333em;
+}
+
+.services-header {
+    min-width: 225px;
+  padding: .5em 0;
+}
+
+[data-title="ux-design"]:after {
+  content: "\e60b";
+}
+
+[data-title="laboratory"]:after {
+  content: "\e600";
+}
+
+[data-title="development"]:after {
+  content: "\e601";
+}
+
+.services-header:after {
+  display: block;
+  width: 2em;
+  height: 2em;
+  margin: .4em auto 0;
+  padding-top: .5em;
+  padding-bottom: .5em;
+  font-family: 'ico-fonts';
+  font-size: 2em;
+  text-align: center;
+  color: #2e2e2e;
+  background-color: #fff;
+  border-radius: 50%;
+}
+
+@media screen and (min-width: 50em) {
+  .services-category-container {
+    display: -webkit-flex;
+    display: flex;
+    -webkit-flex-wrap: wrap;
+    flex-wrap: wrap;
+    -webkit-justify-content: space-around;
+    justify-content: space-around;
+    -webkit-align-content: center;
+    align-content: center;
+  }
+
+  .services-text,
+  .contact-text {
+    padding: 1em;
+  }
+
+  .services-category {
+    width: 33%;
+  }
+}
+
+
+
+/*
+PRICING COMPONENTS
+*/
+
+[class^="pricing"] {
+  text-align: center;
+}
+
+.pricing-tables {
+  width: 90%;
+  min-width: 220px;
+  margin: 1em auto;
+}
+
+.pricing-header {
+  padding: .5em .2em;
+  font-size: 1.5em;
+  font-weight: lighter;
+  letter-spacing: .1em;
+  background-color: #7e3319;
+}
+
+li[class^="pricing"] {
+  padding: 1em;
+  background-color: #fff;
+  border-top: 1px solid #fd6632;
+}
+
+.pricing-plan {
+  color: #fd6632;
+}
+
+.pricing-cost {
+  font-size: 1.333em;
+  font-weight: bold;
+}
+
+.pricing-stats {
+  color: #2e2e2e;
+}
+
+.pricing-button-container {
+  padding: 1.2em;
+  background-color: #be4c25;
+}
+
+.pricing-button {
+  width: 90%;
+  margin: 0 auto;
+  padding: .85em 1.35em;
+  background-color: #fd6632;
+}
+
+@media screen and (min-width: 32em) {
+  .pricing-tables {
+    display: inline-block;
+    width: 45%;
+  }
+}
+
+@media screen and (min-width: 50em) {
+  .pricing {
+    display: -webkit-flex;
+    display: flex;
+    -webkit-flex-wrap: wrap;
+    flex-wrap: wrap;
+    -webkit-justify-content: space-between;
+    justify-content: space-between;
+    -webkit-align-content: center;
+    align-content: center;
+  }
+
+  .pricing-tables {
+    width: 24%;
+  }
+
+  .pricing-button {
+    width: auto;
+  }
+}
+
+
+
+/*
+PORTFOLIO SECTION
+*/
+
+.portfolio-container {
```

Comment:
This block and `.contact-section` share a lot of properties (all but the `background-color`, `max-width`, and `padding`). I'd consolidate them.